app/src/controllers/shop/ShopController.ts

```diff
@@ -133,6 +134,7 @@ const ShopController: ShopControllerInterface = {
     const reservations = await ReservationService.fetchShopReservations(user, shop.id, reservationLimit)
     const menus = await MenuService.fetchShopMenus(user, shop.id, menuLimit)
     const users = await UserService.fetchUsersByIds(reservations.map(r => r.clientId))
+    const totalSales = await ReservationService.getTotalsalesForShop(id)
```

Comment:
for the sake of using the same value as others, i think it looks cleaner if you use shop.id